server/models/interfaces.ts

```diff
@@ -66,3 +66,9 @@ export interface AnomalyResults {
   anomalies: Anomaly[];
   featureData: { [key: string]: FeatureData[] };
 }
+
+export type InitProgress = {
+  percentageStr: string;
+  estimatedMinutesLeft: number;
+  neededShingles: number;
+};
```

Comment:
Minor: do you know if there's a way to have global vars/objs accessible by both the client and server side, to prevent duplicate code and maintenance?

Reply:
Since Kibana plugin helper doesn't copy file in client side to artifact, I guess server side is the only place for such duplicate code. Will verify it and come back with result soon.

Reply:
removed duplicate code and let client side use such type/constants defined in server side.